dsgrid/config/dimensions.py

```diff
@@ -457,7 +456,7 @@ def check_times(cls, ranges, values):
         return _check_time_ranges(ranges, values["str_format"], values["frequency"])
 
     def is_time_zone_required_in_geography(self):
-        return self.timezone == TimeZone.LOCAL
+        return False
```

Comment:
`return self.timezone == TimeZone.NONE`

If we are going down this path, fix `check_timezone()`